extensions/gossip/state/state.go

```diff
@@ -60,7 +60,7 @@ func AddBlockHandler(publisher api.BlockPublisher) {
 }
 
 //NewGossipStateProviderExtension returns new GossipStateProvider Extension implementation
-func NewGossipStateProviderExtension(chainID string, mediator GossipServiceMediator, support *api.Support) GossipStateProviderExtension {
+func NewGossipStateProviderExtension(chainID string, mediator GossipServiceMediator, support *api.Support, blockingMode bool) GossipStateProviderExtension {
```

Comment:
inconsistent variable naming. In other file bool variable has "is" suffix (https://github.com/trustbloc/fabric-mod/pull/107/files#diff-4311ed8b674c06e0800cfc6739e11c5eR19) and this one doesn't. Preferred way would be to name bool variables without suffix.

Reply:
This is existing flag from fabric, maintained same variable name to avoid confusion.